resources/views/admin/role/index.blade.php

```diff
@@ -102,17 +103,48 @@
 @endsection
 @section('script')
     <script type="text/javascript">
-        $('.btn.btn-primary.btn-add-role').click(function (e) {
+        $('.btn.btn-primary.btn-add-form').click(function () {
+            $(".modal-body").html(
+                    `<form>
+                    <div class="form-group">
+                      <label for="recipient-name" class="col-form-label">Name</label>
+                      <input type="text" name="name" class="form-control" id="recipient-name">
+                      <span class="error-form text-danger"></span>
+                    </div>
+                    <button type="button" class="btn btn-primary btn-add-role">Submit</button>
+                  </form>`
+                );
+            $("#myModal").modal('show');
+        });
+
+        $("body").on("click", ".btn.btn-primary.btn-add-role", function (e) {
             e.preventDefault();
             let domForm = $(this).closest('form');
             $.ajax({
                 url: "{{ route('admin.roles.store') }}",
                 data: domForm.serialize(),
                 method: "POST",
             }).done(function (results) {
-                console.log(results);
                 $("#myModal").modal('hide');
-                domForm.reset();
+                $(".table-list-role").append(`
+                    <tr class="data-role-${results.id}">
```

Comment:
https://ourcodeworld.com/articles/read/232/how-to-render-a-view-and-save-its-html-content-in-a-variable-in-laravel

Sử dụng render html trong laravel